momento-sdk/src/main/java/momento/sdk/responses/cache/ttl/ItemGetTtlResponse.java

```diff
@@ -0,0 +1,74 @@
+package momento.sdk.responses.cache.ttl;
+
+import java.time.Duration;
+import momento.sdk.exceptions.SdkException;
+import momento.sdk.internal.StringHelpers;
+
+/**
+ * Parent response type for a cache itemGetTtl request. The response object is resolved to a
+ * type-safe object of one of the following subtypes: {Hit}, {Miss}, {Error}
+ */
+public interface ItemGetTtlResponse {
+
+  /** A successful ≈ operation for a key that exists. */
+  class Hit implements ItemGetTtlResponse {
+    private final Duration ttl;
+
+    /**
+     * Constructs a cache itemGetTtl hit with a ttl value.
+     *
+     * @param ttl the retrieved ttl
+     */
+    public Hit(Duration ttl) {
+      this.ttl = ttl;
+    }
+
+    /**
+     * Gets the retrieved ttl as a Duration.
+     *
+     * @return the ttl.
+     */
+    public Duration ttl() {
+      return ttl;
+    }
+
+    /**
+     * Gets the retrieved ttl as milliseconds since epoch
```

Comment:
is this actually since epoch? I thought it was remaining TTL millis; that's how it is documented in JS SDK?

Reply:
Ah yeah you're right good catch. Just typed it along

Reply:
@cprice404 what do you think about calling this helper method `remainingTtlMillis` and the Duration one as `remainingTtl`

Reply:
uh thats how it is in JS SDk :)